fiftyone/operators/executor.py

```
@@ -993,6 +1037,19 @@ def group_slice(self):
         """The current group slice of the view (if any)."""
         return self.request_params.get("group_slice", None)
 
+    @property
+    def _active_media_type(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_active_media_type`

(ANN202)